src/Fixer/Whitespace/BlankLineBeforeControlStatementFixer.php

```diff
@@ -22,30 +22,23 @@
 use PhpCsFixer\FixerDefinition\FixerDefinition;
 use PhpCsFixer\Tokenizer\Token;
 use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\Tokenizer\TokensAnalyzer;
 
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  * @author Andreas Möller <am@localheinz.com>
+ * @author SpacePossum
  */
 final class BlankLineBeforeControlStatementFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface, WhitespacesAwareFixerInterface
```

Comment:
`include` etc were added for some reason. do we find this reason valid? if so, then I would rename the fixer